README.md

```diff
@@ -182,6 +184,11 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |15|meetings-audio-state|This is an sample tab (side panel) application which shows how to mute/unmute Teams meeting audio using toggle Incoming Client Audio API.||[View][meetings-audio-state#nodejs] ![toolkit-icon](assets/toolkit-icon.png)|
 |16|meetings-context-app|This sample shows the contents of meeting tab context object in a meeting tab and using bot's meeting API, meeting participant details and meeting details are sent to user.|[View][meetings-context-app#csharp]|[View][meetings-context-app#nodejs] ![toolkit-icon](assets/toolkit-icon.png)|
 
+## [Actions Preview](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+|    | Sample Name    | Description                                                                      | C#    | JavaScript      |App Manifest|
+|:--:|:---------------|:---------------------------------------------------------------------------------|:------|:----------------|:-------|
+|1|Actions across Microsoft 365 | This sample contains private preview documentation for (Microsoft 365 extended Teams apps) Actions in Microsoft 365 app(microsoft365.com).| |[View][m365-actions-preview#js] ![toolkit-icon](assets/toolkit-icon.png) 
```

Comment:
Change "private preview" to "preview", since this is a public preview now

Reply:
Fixed